### ISSUE TITLE
The emails that display under profile photo and in my account settings are different.

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#13131]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
Pre condition: User has added research@expensify.com as secondary login
1. Go to Settings and check the email address
2. Go to Profile and check the email address

## Expected Result:
Should be same primary email address displayed

## Actual Result:
Under profile section shows one of the secondary email address

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- Web

**Version Number:** 
**Reproducible in staging?:** Need reproduction
**Reproducible in production?:** Need reproduction
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 
![image (3)](https://user-images.githubusercontent.com/43996225/204563701-4a3251a1-2f13-479e-a0eb-b2045b377033.png)
![image (2)](https://user-images.githubusercontent.com/43996225/204563707-83a625b1-224d-4d63-94c9-950bb06476d0.png)

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @flaviadefaria 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1669369030664359

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by bfitzexpensify** [comment]
@Beamanator tagging you in for the above since you mentioned in Slack that this likely came from https://github.com/Expensify/App/pull/12919

**Comment by twisterdotcom** [comment]
I recreated this today in my account ted@expensify.com:

<img width="1234" alt="image" src="https://user-images.githubusercontent.com/9133401/204789200-a9d5d756-145e-4493-acde-a5910a089bb3.png">

<img width="385" alt="image" src="https://user-images.githubusercontent.com/9133401/204789227-d9e7e330-634c-4ae9-b900-ba6020767f98.png">

<img width="357" alt="image" src="https://user-images.githubusercontent.com/9133401/204789377-29cab266-f3bb-4d10-8172-2ec7c01428f1.png">

**Comment by Beamanator** [comment]
Huh well it looks like i've only got my phone number secondary login stored in Onyx, which makes sense that's why it's the only thing showing on that page :'(

<img width="466" alt="Screen Shot 2022-11-30 at 2 20 55 PM" src="https://user-images.githubusercontent.com/3885503/204794947-1f76c6e9-1456-470e-8a10-b9247fe01649.png">

**Comment by Beamanator** [comment]
Query here: https://github.com/Expensify/Expensify/issues/245563#issuecomment-1332096245

**Comment by Beamanator** [comment]
So looks like the database has the correct dataaaaaa... (see [slack](https://expensify.slack.com/archives/C03TQ48KC/p1669812819294219?thread_ts=1669812705.492589&cid=C03TQ48KC))

**Comment by Beamanator** [comment]
Ok I figured out the problem - it's a backend issue

**Comment by Beamanator** [comment]
Ugg this is going to require 3 PRs
1. App PR accepting new loginList format returned from the backend
2. Backend PR updating the loginList format
3. App PR cleaning up the old loginList format handling

**Comment by joekaufmanexpensify** [comment]
Going to re-open this one for us to do a BZ checklist. Even though this was just a web PR, I feel like it's a good idea to do one as this issue affected app. 

I originally posted the BZ checklist on a [related issue ](https://github.com/Expensify/App/issues/13350#issuecomment-1359333223), but @NikkiWines and I decided it should actually be done here as the other issue was really a duplicate of this one, and the PR there just adjusted the PR from here, which actually fixed the issue,

**Comment by joekaufmanexpensify** [comment]
BugZero Checklist: The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

 [@Beamanator ] The PR that introduced the bug has been identified. Link to the PR:
 [@Beamanator ] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
 [@Beamanator] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
 [@bfitzexpensify ] A [regression test](https://expensify.testrail.io/index.php?/dashboard) has been added or updated so that the same bug will not reach production again. Link to the GH issue for creating the test here:

**Comment by Beamanator** [comment]
@joekaufmanexpensify @bfitzexpensify I filled out the first 3 checkboxes above

For the regression test, I think all we'd need is:
1. Create a user w/ multiple logins using the same email address
    - Example: Login with `yourself@gmail.com` via Google sign in AND with `yourself@gmail.com` via a password (Note: you can only set up google sign in in OldDot for now)
2. Navigate to Settings -> Profile page
3. Verify your "Email address" and "Phone number" show the correct information

**Comment by bfitzexpensify** [comment]
Agreed, posted in #bug-zero [here](https://expensify.slack.com/archives/C01SKUP7QR0/p1671631414523159), I think those instructions are 👌

**Comment by bfitzexpensify** [comment]
OK, we're all done here!
